test/e2e/storage/utils/utils.go

```diff
@@ -61,44 +62,47 @@ const (
 	podSecurityPolicyPrivilegedClusterRoleName = "e2e-test-privileged-psp"
 )
 
-// PodExec wraps RunKubectl to execute a bash cmd in target pod
-func PodExec(pod *v1.Pod, bashExec string) (string, error) {
-	return framework.RunKubectl("exec", fmt.Sprintf("--namespace=%s", pod.Namespace), pod.Name, "--", "/bin/sh", "-c", bashExec)
+// PodExec runs podClient.ExecCommandInContainerWithFullOutput to execute a shell cmd in target pod
+func PodExec(f *framework.Framework, pod *v1.Pod, shExec string) (string, error) {
+	clientSet := f.ClientSet
+	podClient := f.PodClient()
+	stdout, _, err := podClient.ExecCommandInContainerWithFullOutput(pod.Name, pod.Spec.Containers[0].Name, pod.Namespace, clientSet, "/bin/sh", "-c", shExec)
```

Comment:
I think we can use `f.ExecCommandInContainerWithFullOutput` directly.

Reply:
indeed, seems it can, thanks for pointing that out. I changed to that call and patch set became smaller as two added-cloned functions are not needed any more.
Rebased also to HEAD of master and force-pushed.

Reply:
@cofyc: is it okay now?

Reply:
yes, looks good to me (except one nit you pointed [here](https://github.com/kubernetes/kubernetes/pull/84042#discussion_r344926110))